.golangci.yml

```diff
@@ -18,6 +18,7 @@ linters:
     - scopelint
     - golint
     - maligned
+    - exhaustruct
```

Comment:
This rule seem to be too aggressive for me. https://github.com/ubie-oss/sendgrid-stats-exporter/runs/6426198927?check_suite_focus=true